src/main/server.ts

```diff
@@ -33,26 +36,20 @@ function getSSLOptions (): any {
   }
 }
 
-function forkListenerProcesses (numberOfCores: number) {
-  for (let i = 0; i < numberOfCores; i++) {
-    cluster.fork()
+function applicationPort (): any {
```

Comment:
probably should have a comment about why this is `any` and if you have time on Monday would be awesome to look at sending a PR for the express types

Reply:
Yes, will add a comment and see about the PR.